addons/escoria-core/game/core-scripts/esc/commands/camera_set_pos.gd

```diff
@@ -24,6 +24,11 @@ func configure() -> ESCCommandArgumentDescriptor:
 	)
 
 
+# Validate whether the given arguments match the command descriptor
```

Comment:
Hm. I'd argue that this implementation isn't actually needed because it is called through inheritance or am I missing something?